platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/status/PlatformStatusManager.java

```diff
@@ -75,7 +75,7 @@ public PlatformStatusManager(
 
         this.queue = new QueueThreadConfiguration<PlatformStatusAction>(threadManager)
                 .setComponent("platform")
-                .setThreadName("status-state-machine")
+                .setThreadName("status_state_machine")
```

Comment:
This does not only change the metrics name. I do not know if platform uses the name somehow else. cc @cody-littley @lpetrovic05

Reply:
I checked with @poulok before pushing the change... the naming is less readable but at least consistent with the metrics created under the hood.

Reply:
Thread names are purely used for understanding thread dumps and for logging. There are no hard dependencies I'm aware of. Anyways, we are very soon going to completely drop support for this style of threads, so naming is not super important.